dockerStart.sh

```diff
@@ -1,3 +1,18 @@
-#!/usr/bin/env sh
+#!/bin/bash -l
+set -eu
 
-exec pdm run venmo-auto-cashout
+# export all env vars to a file which is automatically sourced inside the cron subshell
+# https://gist.github.com/athlan/b6f09977e2f5cf20840ef61ca3cda932
+printenv | awk -F= '{print "export " "\""$1"\"""=""\""$2"\"" }' >>/etc/profile
+
+if [[ ${SCHEDULE} == "NONE" ]]; then
+	pdm run venmo-auto-cashout
```

Comment:
Can this be 
```suggestion
	exec pdm run venmo-auto-cashout
```

and then we don't need the `else` case

Reply:
not following: why wouldn't we need the else case here? The else is for running this on a schedule.